store/blocks.json

```diff
@@ -8,5 +8,10 @@
       "total-products",
       "order-by"
     ]
+  },
+  "filter-navigator":{
```

Comment:
this is not correct, this would make `shop-review-widget` a default block for filter-navigator.

Reply:
What should I do to correct this so that the widget is allowed in the filter-navigator blocks? 

I've tried changing "blocks" to "allowed" and removing "filter-navigator" section from the blocks.json file completely. The former gives me an error stating that "allowed" is an unsupported field when I try to link the project and the latter results in my shop-review-widget not displaying inside the filter-navigator block in my workspace despite being defined as a block within my store-theme like this;

"search-result": {
    "blocks": [
      "filter-navigator",
      "gallery",
      "not-found",
      "breadcrumb",
      "total-products",
      "order-by"
    ]
  },
  "filter-navigator":{
    "blocks":[
      "shop-review-widget.shopper-approved-widget"
    ]
  },

Reply:
Not sure why your store-theme isn't working, but you've defined the block as allowed in interfaces.json and that should be all you need to do. The blocks.json in this repo is basically just an "initial state" and would typically be overridden by the store-theme anyway.